src/sentry/grouping/strategies/newstyle.py

```diff
@@ -48,6 +48,20 @@
     "colno",
 ]
 
+# Ignore all those kinds of exception types as they are platform specific
+# For example there can be crashes with EXC_ACCESS_VIOLATION_* on Windows with
+# the same exact stacktrace as a crash with EXC_BAD_ACCESS on macOS
+_discard_native_error_code_re = re.compile(
+    r"""(
+    ^EXC_ |
+    ^EXCEPTION_ |
+    ^SIG |
+    ^KERN_ |
+    ^ILL_
+)""",
```

Comment:
I would probably change to `(what you have)[A-Z0-9_]+$` to feel more confident we don't accidentally discard stuff from users.

Reply:
need to add `[ /]` to that set because of `EXC_BAD_ACCESS / ...`

Reply:
Ah yeah :-/ That's surprisingly annoying that it gets reported like that.

Reply:
also added `x` for `0x00`...